pom.xml

```diff
@@ -10,7 +10,7 @@
     <properties>
         <fabric8-version>4.13.2</fabric8-version>
         <org-json-version>20240303</org-json-version>
-        <jetty-version>9.4.55.v20240627</jetty-version>
+        <jetty-version>9.4.56.v20240826</jetty-version>
```

Comment:
Can we check with `12.0.15` as that is the latest?

Reply:
With jetty 12.0.15 & 12.0.3 versions, I get the below error:

```
[ERROR] Failed to execute goal on project autotune: Could not resolve dependencies for project org.autotune:autotune:jar:0.1.1
[ERROR] dependency: org.eclipse.jetty:jetty-servlets:jar:12.0.15 (compile)
[ERROR] 	Could not find artifact org.eclipse.jetty:jetty-servlets:jar:12.0.15 in central (https://repo.maven.apache.org/maven2)
[ERROR] dependency: org.eclipse.jetty:jetty-servlet:jar:12.0.15 (compile)
[ERROR] 	Could not find artifact org.eclipse.jetty:jetty-servlet:jar:12.0.15 in central (https://repo.maven.apache.org/maven2)
```

Tried 11.0.24, but got the below compilation errors
```
[ERROR] /home/autotune/src/autotune/src/main/java/com/autotune/experimentManager/handler/LoadValidationHandler.java:[32,21] package javax.servlet does not exist
[ERROR] /home/autotune/src/autotune/src/main/java/com/autotune/experimentManager/handler/eminterface/EMHandlerInterface.java:[25,21] package javax.servlet does not exist
[ERROR] /home/autotune/src/autotune/src/main/java/com/autotune/experimentManager/handler/LoadValidationHandler.java:[42,187] cannot find symbol
  symbol:   class ServletContext
  location: class com.autotune.experimentManager.handler.LoadValidationHandler
[ERROR] /home/autotune/src/autotune/src/main/java/com/autotune/experimentManager/handler/eminterface/EMHandlerInterface.java:[35,65] cannot find symbol
  symbol:   class ServletContext
  location: interface com.autotune.experimentManager.handler.eminterface.EMHandlerInterface
[ERROR] /home/autotune/src/autotune/src/main/java/com/autotune/analyzer/utils/AnalyzerErrorConstants.java:[25,26] package javax.servlet.http does not exist
[ERROR] /home/autotune/src/autotune/src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileValidation.java:[33,26] package javax.servlet.http does not exist
[ERROR] /home/autotune/src/autotune/src/main/java/com/autotune/analyzer/services/ListExperiments.java:[49,21] package javax.servlet does not exist
```

Checking with 10.0.24